frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/Step2Geometry.vue

```diff
@@ -389,6 +397,18 @@ export default class Step2Geometry extends Vue {
       : 1;
   }
 
+  private updateHabitability(): void {
+    // Door(s) >= 90cm wide INPUT 5 habitability
+    const { doors_dimensions } = this.localShelter.geometry;
```

Comment:
const doors_dimensions = this.localShelter.geometry.doors_dimensions; ?

Reply:
Je trouve ça plus lisible avec { doors_dimensions } c'est plus DRY